datasets/qqp_checklist/qqp_checklist.py

```diff
@@ -0,0 +1,115 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A CheckList suite for Quora Question Pair"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import json
+import logging
+import os
+
+import nlp
+from checklist.test_suite import TestSuite
+
+
+_CITATION = """\
+ @inproceedings{checklist:acl20},
+ author = {Marco Tulio Ribeiro and Tongshuang Wu and Carlos Guestrin and Sameer Singh},
+ title = {Beyond Accuracy: Behavioral Testing of NLP models with CheckList},
+ booktitle = {Association for Computational Linguistics (ACL)},
+ year = {2020}
+"""
+
+_DESCRIPTION = """\
+A CheckList for Quora Question Pair.
+Predictions: should be integers, where:
+  - 0: non-duplicate
+  - 1: duplicate
+Confidences: should be list(float) of length 2, with prediction probabilities
+for non-duplicate and duplicate(respectively)
+
+Test names for Table 2 in the paper:
+['Modifier: adj',  'How can I become more {synonym}?', 'Replace synonyms in real pairs', 'How can I become more X = How can I become less antonym(X)', 'add one typo', '(q, paraphrase(q))',  'Change same name in both questions',  'Change first and last name in one of the questions', 'Keep entitites, fill in with gibberish', 'Is person X != Did person use to be X',   'Is it {ok, dangerous, ...} to {smoke, rest, ...} after != before', "What was person's life before becoming X != What was person's life after becoming X", 'How can I become a X person != How can I become a person who is not X', 'How can I become a X person == How can I become a person who is not antonym(X)', 'Simple coref: he and she', 'Simple coref: his and her',  'Order does not matter for comparison', 'Order does not matter for symmetric relations', 'Order does matter for asymmetric relations',  'traditional SRL: active / passive swap with people', 'traditional SRL: wrong active / passive swap with people', 'Symmetry: f(a, b) = f(b, a)', 'Testing implications']
+
+Use with nlp.checklist.CheckListSuite
+"""
+
+_URL = "https://github.com/marcotcr/checklist/raw/master/release_suites/qqp_suite.tar.gz"
+_SUITE_NAME = "qqp_suite.pkl"
+
+
+class QqpCheckListConfig(nlp.BuilderConfig):
+    """ BuilderConfig for NewDataset"""
+
+    def __init__(self, url=_URL, suite_name=_SUITE_NAME, **kwargs):
+        """
+
+        Args:
+            url: .tar.gz file url
+            suite_name: name of the pickle file archived in url
+            **kwargs: keyword arguments forwarded to super.
+        """
+        self.url = url
+        self.suite_name = suite_name
+        super(QqpCheckListConfig, self).__init__(**kwargs)
+
+
+class QqpCheckList(nlp.GeneratorBasedBuilder):
+    """TODO: Short description of my dataset."""
+
+    VERSION = nlp.Version("1.1.0")
+
+    BUILDER_CONFIG_CLASS = QqpCheckListConfig
+    BUILDER_CONFIGS = [QqpCheckListConfig(_URL, _SUITE_NAME, version=VERSION, name="qqp_checklist_config")]
+
```

Comment:
```suggestion

    test_dummy_data = False

```

Will skip automatically triggered dataset test (currently failing because the CI doesn't have `checklists` installed)